tests/unittest_protocols.py

```diff
@@ -0,0 +1,21 @@
+from typing_extensions import Protocol
+from screenpy.protocols import Answerable, Forgettable, Performable, ErrorKeeper
+
+
+class Describable(Protocol):
+    def describe(self) -> str:
+        ...
+
+class Question(Answerable, Describable):
+    ...
+
+class ErrorQuestion(Answerable, Describable, ErrorKeeper):
+    ...
+
+class Action(Performable, Describable):
+    ...
+
+class Ability(Forgettable):
+    ...
```

Comment:
This is an interesting approach. I think i like it!

However i think we have a `Describable` protocol now (from you!), so i think the one on line 5 is not necessary.

Reply:
Weird, that's an obvious one. I'm not sure why I have this here. I'll update it.